### DISCUSSION TITLE
Fooocus future

### DISCUSSION THREAD

**Discussion by E2GO** [mashb1t/Fooocus#76] [Q&A] [answered]
Hey @mashb1t! First of all thanks for a nice Fooocus fork and all support you give to it! I've heard some rumors from here and there on not so bright future for Fooocus and just wanted to know what the future for your fork or for main? Your thought and plans may be) Thanks in advance!

**Comment by mashb1t** [comment, marked as answer]
Hey @E2GO, you can find more in https://github.com/lllyasviel/Fooocus/discussions/3504 and specifically concerning this fork in https://github.com/lllyasviel/Fooocus/discussions/3504#discussioncomment-10313526.
